app/config/index.js

```diff
@@ -1,38 +1,29 @@
-const Joi = require('joi')
-const dbConfig = require('./db')
-const messageQueueConfig = require('./message-queue')
+import joi from 'joi'
+import dbConfig from './db.js'
+import messageQueueConfig from './message-queue.js'
 
-const schema = Joi.object({
-  port: Joi.number(),
-  env: Joi.string().valid('development', 'test', 'production').default('development'),
-  isDev: Joi.boolean().default(false),
-  termsAndConditionsUrl: Joi.string().default('#'),
-  applyServiceUri: Joi.string().default('#'),
-  claimServiceUri: Joi.string().default('#'),
-  endemics: Joi.object({
-    enabled: Joi.boolean().default(false)
-  })
+const schema = joi.object({
+  port: joi.number(),
+  env: joi.string().valid('development', 'test', 'production').default('development'),
+  isDev: joi.boolean().default(false),
+  termsAndConditionsUrl: joi.string().default('#'),
```

Comment:
I don't think this app needs any of terms and conditions Url, applyService Url or claim Service Url, it doesn;t have use for any of those

Reply:
Cool! Removed them.